src/analysisd/decoders/syscheck.c

```diff
@@ -1456,6 +1474,9 @@ static int fim_generate_alert(Eventinfo *lf, char *mode, char *event_type,
             //lf->fields[FIM_SYM_PATH].value
     );
 
+    cJSON_Delete(item);
```

Comment:
`item` is an iterator pointer, we must not delete it. `cJSON_Delete(tmp)` will fully delete the structure.